### PR TITLE
Update harbour-determinant-es.ts

### PR DESCRIPTION
**Pull request by carlosgonz0**

### CODE REVIEW

File: translations/harbour-determinant-es.ts [hunk 1]
```diff
@@ -91,7 +91,7 @@
     <name>MessageInput</name>
     <message>
         <source>Message</source>
-        <translation>Mensaje</translation>
+        <translation>A b c</translation>
```
Comment (R1tschY) [inline translations/harbour-determinant-es.ts:94]: This is probably a mistake

File: translations/harbour-determinant-es.ts [hunk 2]
```diff
@@ -212,43 +212,43 @@
     </message>
     <message>
         <source>Aliases</source>
-        <translation>Alias</translation>
+        <translation>Alias:</translation>
```
Comment (R1tschY) [inline translations/harbour-determinant-es.ts]: colons are not intended

File: translations/harbour-determinant-es.ts [hunk 3]
```diff
@@ -267,11 +267,11 @@
     </message>
     <message>
         <source>Invitation</source>
-        <translation type="unfinished"></translation>
+        <translation>Invitación </translation>
```
Comment (R1tschY) [inline translations/harbour-determinant-es.ts:270]: trailing whitespace

File: translations/harbour-determinant-es.ts [hunk 4]
```diff
@@ -267,11 +267,11 @@
...
     </message>
     <message>
         <source>Logout</source>
-        <translation type="unfinished"></translation>
+        <translation>Cerrar sesión </translation>
```
Comment (R1tschY) [inline translations/harbour-determinant-es.ts]: trailing whitespace

**Review by R1tschY — CHANGES_REQUESTED** [review]

File: translations/harbour-determinant-es.ts (hunk 1 above)
Comment (carlosgonz0) [reply translations/harbour-determinant-es.ts:94]: A b c is good, it is my own form of spanish. Fernschreiber has same A b c so i want matching....

File: translations/harbour-determinant-es.ts (hunk 2 above)
Comment (carlosgonz0) [reply translations/harbour-determinant-es.ts]: I will  delete the colons

File: translations/harbour-determinant-es.ts (hunk 4 above)
Comment (carlosgonz0) [reply translations/harbour-determinant-es.ts]: What wrong here? you mean without space between words?

Comment (R1tschY) [reply translations/harbour-determinant-es.ts]: I mean the space at the end. It should be removed.

**Pull request merged**